terraform/google_gke/main.tf

```diff
@@ -20,6 +20,17 @@ resource "google_container_cluster" "default" {
   release_channel {
     channel = "REGULAR"
   }
+  maintenance_policy {
+    daily_maintenance_window {
+      start_time = "22:00"
```

Comment:
Like above these are wall clock times in 24h format without timezone reference. I guess it is supposed to be GMT because it runs in the cloud but we'll never know. A comment or even better a variable that encodes the unit would be perfect.

Same goes for all timing information in this change.

I also come to wonder where these variables are used because this change is the only reference in this PR that regards these variables. Are they already in the code base?

Reply:
This is a terraform module which we use in our mn-ces-gke-example and our coder instance. The module is referenced there by a git url.